tasks/AccelerationController.cpp

```diff
@@ -25,16 +25,17 @@ bool AccelerationController::configureHook()
     if (! AccelerationControllerBase::configureHook()){
         return false;
     }
-    thrusterMatrix = _matrix.get().transpose();
+    thrusterMatrix = _matrix.get();
+    numberOfThrusters = thrusterMatrix.cols();
```

Comment:
don't store numberOfThrusters as a class attribute, it duplicates information. If it clarifies the code in this method (as it seems), just cache it as local variable(s). Otherwise, you can also make it a method which would extract the information from the matrix.